hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -534,6 +542,52 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
       blockTokenMgr = createBlockTokenSecretManager(configuration);
     }
 
+    // Get S3 multi-tenancy enabled flag
```

Comment:
Can we do this in a helper method? OM constructor is already quite large.

Reply:
Moved to a static method under `OMMultiTenantManager`